content/webapp/components/GuideStopCard/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { FunctionComponent } from 'react';
+import { getCrop, ImageType } from '@weco/common/model/image';
+import PrismicImage from '@weco/common/views/components/PrismicImage/PrismicImage';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import Space from '@weco/common/views/components/styled/Space';
+import { grid } from '@weco/common/utils/classnames';
+import { threeUpGridSizesMap } from '@weco/content/components/Body/GridFactory';
+import ImagePlaceholder from '@weco/content/components/ImagePlaceholder/ImagePlaceholder';
+import { secondsToHoursMinutesAndSeconds } from '@weco/common/utils/format-time';
+import type { ColorSelection } from '@weco/content/types/color-selections';
+
+import {
+  map,
+  audio as audioIcon,
+  video as videoIcon,
+} from '@weco/common/icons';
+import {
+  CardOuter,
+  CardTitle,
+  CardBody,
+  CardImageWrapper,
+} from '@weco/content/components/Card/Card';
+
+type Props = {
+  link?: string;
+  number?: number;
+  totalStops: number;
+  title: string;
+  duration?: number;
+  type: 'audio' | 'video';
+  image?: ImageType;
+};
+
+const GuideStopCard: FunctionComponent<Props> = ({
+  link,
+  number,
+  totalStops,
+  title,
+  duration,
+  type,
+  image,
+}) => {
+  const durationInSeconds = duration && Math.round(duration * 60);
+  const durationInMinutesAndSeconds = durationInSeconds
+    ? secondsToHoursMinutesAndSeconds(durationInSeconds)
+    : undefined;
+  const croppedImage = getCrop(image, '16:9');
+  const placeholderBackgroundColor = (stopNumber: number): ColorSelection => {
```

Comment:
I think we'll probably want to have the same behaviour for the placeholders on the individual stop pages so this probably isn't where this function will end up, but probably ok here for now (since those stop pages don't exist yet)

Reply:
I love this